parser/build.gradle

```diff
@@ -27,15 +27,15 @@ repositories {
 
 dependencies {
     api "org.jetbrains.kotlin:kotlin-stdlib:$kotlinVersion"
-    api "com.xenomachina:kotlin-argparser:2.0.7"
+    implementation("com.github.ajalt.clikt:clikt:3.1.0")
```

Comment:
```suggestion
    implementation "com.github.ajalt.clikt:clikt:3.1.0"
```

Reply:
Do we want to standardize double quotes and single quote as well?